src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/SessionTests.cs

```diff
@@ -203,4 +204,347 @@ static ChannelFactory<T> CreateChannelFactoryHelper<T>(string url)
         binding.Security.Mode = SecurityMode.None;
         return new ChannelFactory<T>(binding, helloEndpoint);
     }
+
+    // Without CallbackBehaviorAttribute (and ConcurrencyMode = ConcurrencyMode.Multiple) support
+    // a few duplex session tests are not going to be able to call the service from within our callback. 
+    // This is tracked by issue #1959
+
+    //[CallbackBehavior(ConcurrencyMode = ConcurrencyMode.Multiple)]
+    public class SessionTestsDuplexCallback : ISessionTestsDuplexCallback
```

Comment:
Files ending in "Tests" as a general rule should only contain tests. Please put all helper and/or common code in a separate file such as SessionTestHelpers.cs or SessionTestCommon.cs